code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -196,7 +203,7 @@
 /mob/camera/aiEye/remote
 	name = "Inactive Camera Eye"
 	ai_detector_visible = FALSE
-	var/sprint = 10
+	var/sprint = 5
```

Comment:
Explai this change

Reply:
mistake, i was just testing if that var had any impact. It has none